src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,233 @@
+<script lang="ts">
+  import type { QuizData } from '$lib/constants/quiz';
+  import LeftArrow from '../icons/leftArrow.svelte';
+  import RightArrow from '../icons/rightArrow.svelte';
```

Comment:
To import common icons, you can use `$lib`.
```suggestion
  import LeftArrow from '$lib/components/icons/left-arrow.svelte';
  import RightArrow from '$lib/components/icons/right-arrow.svelte';
```

Reply:
Done